build_resolvers/lib/src/build_asset_uri_resolver.dart

```diff
@@ -226,7 +243,7 @@ Future<String> packagePath(String package) async {
 
 /// Returns all the directives from a Dart library that can be resolved to an
 /// [AssetId].
-Set<AssetId> _parseDirectives(String content, AssetId from) => HashSet.of(
+Set<AssetId> parseDependencies(String content, AssetId from) => HashSet.of(
```

Comment:
[nit] can this stay private? I don't see any usage in the PR outside this library.

Reply:
Yep just forgot to swap it back after experimenting (initially I just made this public and used it directly).